src/details/ArborX_DetailsHalfTraversal.hpp

```diff
@@ -0,0 +1,88 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
```

Comment:
```suggestion
 * Copyright (c) 2017-2023 by the ArborX authors                            *
```
Here and other files.